browser/src/control/Control.SheetsBar.js

```diff
@@ -69,6 +69,9 @@ L.Control.SheetsBar = L.Control.extend({
 		}
 
 		if (id === 'insertsheet') {
+			if (this.map._docLayer.isAutoFillPopupOpen)
+				this._map.fire('closeautofillpopup');
```

Comment:
Do we need to fire an event here? Maybe we can use objectInstance.openAutoFillPopup()?

Also, 'openautofillpopup' event does the same thing it seems. And i couldn't see its usage.

Reply:
I think you mean ` objectInstance.closeAutoFillPopup()` ?.  I'd like to close `AutoFill` popup here if we already have it open. Could you please provide further explanation on that :) Thanks.